File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Asks user to enter the height between 1 and 8 inclusive
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    // Creates the pyramid
    for (int i = 0; i < height; i++)
    {
        // Creates the empty space to the left
        for (int j = 0; j < height - i - 1; j++)
        {
            printf(" ");
        }
        // The left pyramid structure
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }
        // Creates the spacing b/w left and right blocks
        printf("  ");

        // Creates the right hand blocks
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }

        // Moves down a level
        printf("\n");
    }
};

File: pset6/mario/more/mario.py
from cs50 import get_int

while True:
    height = get_int("height: ")
    if height > 0 and height < 9:
        break

for i in range(height):
    # Prints the spaces on the left
    for j in range(height - i - 1):
        print(" ", end='')
        
    # Prints the left pyramid
    for k in range(i + 1):
        print("#", end='')
        
    # Prints the gap b/w the 2 pyramids
    print("  ", end='')
    
    # Prints the right pyramid
    for l in range(i + 1):
        print("#", end='')
        
    # Brings printer down to next level
    print("")